setup.py

```diff
@@ -56,7 +56,7 @@
 # note: cocotb is not installed properly yet, but we can import it anyway
 # because it's in the current directory. We'll need to change this if we
 # add `install_requires` to the `setup()` call.
-from cocotb._build_libs import get_ext, build_ext
+from build_libs import get_ext, build_ext
```

Comment:
Possibly worth giving it an obvious prefix, either
```suggestion
from cocotb_build_libs import get_ext, build_ext
```
or
```suggestion
from setup_build_libs import get_ext, build_ext
```